_templates/_page_openshift.html.erb

```diff
@@ -0,0 +1,94 @@
+    <script src="<%= File.join(javascripts_path, "respond.proxy.js") %>" type="text/javascript"></script>
+  <![endif]-->
+  <script src="https://assets.openshift.net/content/modernizr.js" type="text/javascript"></script>
+  <script src="https://assets.openshift.net/content/subdomain.js" type="text/javascript"></script>
+  <script src="<%= File.join(javascripts_path, "bootstrap-offcanvas.js") %>" type="text/javascript"></script>
+  <%= render("_templates/_analytics.html.erb", :distro_key => distro_key) %>
+</head>
+<body>
+  <%= render("_templates/_topnav.html.erb", :distro_key => distro_key) %>
+  <div class="container">
+    <p class="toggle-nav visible-xs pull-left">
+      <button class="btn btn-default btn-sm" type="button" data-toggle="offcanvas">Toggle nav</button>
+    </p>
+    <ol class="breadcrumb">
+      <li class="sitename">
+        <a href="<%= site_home_path %>"><%= site_name %></a>
+      </li>
+      <li class="hidden-xs active">
+        <%= breadcrumb_root %>
+      </li>
+      <li class="hidden-xs active">
+        <%= breadcrumb_group %>
+      </li>
+      <%= breadcrumb_subgroup_block %>
+      <li class="hidden-xs active">
+        <%= breadcrumb_topic %>
+      </li>
+    </ol>
+    <div class="row row-offcanvas row-offcanvas-left">
+      <div class="col-xs-8 col-sm-3 col-md-3 sidebar sidebar-offcanvas">
+        <div class="row-fluid">
+          <%= render("_templates/_search.html.erb", :distro_key => distro_key) %>
+        </div>
+        <%= render("_templates/_nav_openshift.html.erb", :navigation => navigation, :group_id => group_id, :topic_id => topic_id, :subgroup_id => subgroup_id, :subtopic_shim => subtopic_shim) %>
+      </div>
+      <div class="col-xs-12 col-sm-9 col-md-9 main">
+        <div class="page-header">
+          <h2><%= article_title %></h2>
+        </div>
+        <%= content %>
+      </div>
+    </div>
+  </div>
+  <script type="text/javascript">
+    /*<![CDATA[*/
+    $(document).ready(function() {
+      $("[id^='topicGroup']").on('show.bs.collapse', function (event) {
+        if (!($(event.target).attr('id').match(/^topicSubGroup/))) {
+          $(this).parent().find("[id^='tgSpan']").toggleClass("fa-angle-right fa-angle-down");
+        }
+      });
+      $("[id^='topicGroup']").on('hide.bs.collapse', function (event) {
+        if (!($(event.target).attr('id').match(/^topicSubGroup/))) {
+          $(this).parent().find("[id^='tgSpan']").toggleClass("fa-angle-right fa-angle-down");
+        }
+      });
+      $("[id^='topicSubGroup']").on('show.bs.collapse', function () {
+        $(this).parent().find("[id^='sgSpan']").toggleClass("fa-caret-right fa-caret-down");
+      });
+      $("[id^='topicSubGroup']").on('hide.bs.collapse', function () {
+        $(this).parent().find("[id^='sgSpan']").toggleClass("fa-caret-right fa-caret-down");
+      });
+    });
+    /*]]>*/
+  </script>
+  <%= render("_templates/_footer.html.erb", :distro_key => distro_key) %>
```

Comment:
Ok, thanks for tracking this down. Not that complicated once you follow along. I'll update.